### PR TITLE
Add l10n it fatturapa notifications

### PR DESCRIPTION
**Pull request by eLBati**
=======================
FatturaPA Notifications
=======================

The module handles notifications linked to fatturaPA files.
http://www.fatturapa.gov.it/export/fatturazione/en/normativa/f-3.htm?l=en
http://www.fatturapa.gov.it/export/fatturazione/it/b-2.htm?l=it

Usage
=====

The module adds 'fatturapa.notification' model and exposes 'parse_xml' method
to be called by other modules.
It also adds the 'Import notification' wizard that allows to load XML
notification files and link them to fatturaPA

================================
Supplier FatturaPA Notifications
================================

This module handles SDI notifications related to supplier invoices (fatturaPA).
See http://fatturapa.gov.it/export/fatturazione/it/b-2.htm

Usage
=====

Within 'fatturaPA' tab of supplier invoice form, you can click on 2 buttons:
 - Accept fatturaPA
 - Reject fatturaPA
This allows to generate the 'notifica esito committente' message
http://fatturapa.gov.it/export/fatturazione/sdi/messaggi/v1.0/IT01234567890_11111_EC_001.xml

### CODE REVIEW

File: l10n_it_fatturapa_in_notifications/__openerp__.py [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+        'l10n_it_fatturapa_in',
+        'l10n_it_fatturapa_notifications',
+    ],
+    "data": [
+        'attachment_view.xml',
+        'wizard/send_notification_view.xml',
+        'account_view.xml',
+    ],
+    "test": [],
+    "demo": [],
```
Comment (tafaRU) [inline l10n_it_fatturapa_in_notifications/__openerp__.py]: Please remove these empty keys

File: l10n_it_fatturapa_in_notifications/tests/data/IT05979361218_002_EC_002.xml [hunk 2]
```diff
@@ -0,0 +1 @@
+<?xml version="1.0" encoding="utf-8"?><ns1:NotificaEsitoCommittente versione="1.0" xmlns:ns1="http://www.fatturapa.gov.it/sdi/messaggi/v1.0"><IdentificativoSdI>5199973</IdentificativoSdI><RiferimentoFattura><NumeroFattura>FT/2015/0007</NumeroFattura><AnnoFattura>2015</AnnoFattura></RiferimentoFattura><Esito>EC01</Esito><Descrizione>accept</Descrizione></ns1:NotificaEsitoCommittente>
```
Comment (tafaRU) [inline l10n_it_fatturapa_in_notifications/tests/data/IT05979361218_002_EC_002.xml]: No newline at end of file

File: l10n_it_fatturapa_notifications/__openerp__.py [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+    "depends": [
+        'l10n_it_fatturapa',
+    ],
+    "data": [
+        'attachment_view.xml',
+        'wizard/import_notification_view.xml',
+        'security/ir.model.access.csv',
+    ],
+    "test": [],
+    "demo": [],
```
Comment (tafaRU) [inline l10n_it_fatturapa_notifications/__openerp__.py]: Same as above

File: l10n_it_fatturapa_in_notifications/__openerp__.py [hunk 4]
```diff
@@ -0,0 +1,101 @@
...
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+{
+    'name': 'Supplier FatturaPA - Notifications',
+    'version': '7.0.1.0.0',
+    'category': 'Localization/Italy',
+    'summary': 'Supplier electronic invoices notifications',
+    'author': 'Agile Business Group',
```
Comment (tafaRU) [inline l10n_it_fatturapa_in_notifications/__openerp__.py]: Please add **Odoo Community Association (OCA)** as raised by Travis build.

File: l10n_it_fatturapa_notifications/__openerp__.py [hunk 5]
```diff
@@ -0,0 +1,99 @@
...
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+{
+    'name': 'FatturaPA - Notifications',
+    'version': '7.0.1.0.0',
+    'category': 'Hidden',
+    'summary': 'Electronic invoices notifications',
+    'author': 'Agile Business Group',
```
Comment (tafaRU) [inline l10n_it_fatturapa_notifications/__openerp__.py]: Same as above.

**Comment by tafaRU** [comment]
Few nitpicking comments otherwise :+1:

File: l10n_it_fatturapa_in_notifications/i18n/it.po [hunk 6]
```diff
@@ -0,0 +1,227 @@
...
+#. module: l10n_it_fatturapa_in_notifications
+#: field:account.invoice,Vehicle_registration:0
+msgid "Veicole Registration"
+msgstr "Data immatricolazione "
+
+#. module: l10n_it_fatturapa_in_notifications
+#: view:wizard.fatturapa.in.accept:0
+#: view:wizard.fatturapa.in.reject:0
+msgid "or"
+msgstr "or"
```
Comment (OpenCode) [inline l10n_it_fatturapa_in_notifications/i18n/it.po]: "or" is "o" in italian :)

File: l10n_it_fatturapa_notifications/i18n/it.po [hunk 7]
```diff
@@ -0,0 +1,172 @@
...
+
+#. module: l10n_it_fatturapa_notifications
+#: help:wizard.fatturapa.import.notification,name:0
+msgid "XML notification file"
+msgstr "File XML di notifica"
+
+#. module: l10n_it_fatturapa_notifications
+#: view:wizard.fatturapa.import.notification:0
+msgid "or"
+msgstr "or"
```
Comment (angedras) [inline l10n_it_fatturapa_notifications/i18n/it.po]: the translation is "o" not "or"

**Comment by eLBati** [comment]
@tafaRU @angedras @OpenCode thanks. done

**Comment by archetipo** [comment]
:+1:

**Comment by tafaRU** [comment]
Runbot build is yellow due to https://github.com/OCA/l10n-italy/issues/161

Considering it's not related to this PR, I proceed with the merge.

**Pull request merged**